pkg/state/state.go

```diff
@@ -213,6 +215,7 @@ type RepositorySpec struct {
 	Keyring         string `yaml:"keyring,omitempty"`
 	PassCredentials bool   `yaml:"passCredentials,omitempty"`
 	SkipTLSVerify   bool   `yaml:"skipTLSVerify,omitempty"`
+	PlainHttp       bool   `yaml:"plainHttp,omitempty"`
```

Comment:
I can't see any code that handle thie field. @hoangelos

Reply:
@hoangelos

Reply:
@yxxhero  I pinged below on where it is.